__tests__/components/editor/PropertyTemplateOutline.test.js

```diff
@@ -21,4 +29,54 @@ describe('<PropertyTemplateOutline />', () => {
   it('Contains a <PropertyTypeRow />', () => {
     expect(wrapper.find(PropertyTypeRow)).toBeTruthy()
   })
+
+  it('has an <PropertyTemplateOutline /> as a child', () => {
+     expect(wrapper.find(PropertyTemplateOutline)).toBeTruthy()
+  })
+
+  it('child PropertyTemplateOutline has an InputLiteral', () => {
+    const childPropertyTemplateOutline = wrapper.find(PropertyTemplateOutline)
+    expect(childPropertyTemplateOutline.find(InputLiteral)).toBeTruthy()
+  })
+})
+
+describe('<PropertyTemplateOutline /> with propertyTemplate Refs', () => {
+  const mockHandleCollapsed = jest.fn()
+  const propertyRtProps = {
+    resourceTemplate: {
+      resourceLabel: "Test Schema Thing CreativeWork",
+      propertyTemplates: [
+        {
+          "propertyLabel": "Notes about the CreativeWork",
+          "remark": "This is a great note",
+          "propertyURI": "http://id.loc.gov/ontologies/bibframe/note",
+          "mandatory": "false",
+          "repeatable": "true",
+          "type": "resource",
+          "resourceTemplates": [],
+          "valueConstraint": {
+            "valueTemplateRefs": [
+              "resourceTemplate:bf2:Note"
+            ],
+            "useValuesFrom": [],
+            "valueDataType": {},
+            "defaults": []
+          }
+        }
+      ]
+    }
+  }
+  const wrapper = mount(<PropertyTemplateOutline {...propertyRtProps}
```

Comment:
Is it possible to write some tests for the other propertyTemplate types (using different `propertyRtProps`) in order to get better test coverage on this component? Otherwise future PRs will need to address this code coverage (or we will have to keep decreasing the threshold). Coverage is down to 76% on this PR and we currently have 75% as the bottom threshold.

Reply:
Thanks @jgreben! I'll add some tests to increase coverage.

Reply:
Hi @jgreben, the latest commit puts code coverage back over 77%